Tuto-GUDHI-cover-complex.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>The upper plot works for me, but not this one. I did not invetigate why</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
mmmh that's weird, the two plots appear on my side, but I waited for the previous cell to finish before plotting. Maybe the plot does not render properly if another cell is running?

Reply:
<p>I tried again and that's ok for me</p>